client/src/components/Projects/ProjectEdit/index.jsx

```diff
@@ -20,14 +21,20 @@ export default class index extends Component {
 
   componentDidMount() {
     // fetch project information by take project id from this.props.match.projectId
-    const projectInf = require("./../../commonComponents/TableMember/projects.json")[0];
-    this.setState({
-      project: projectInf,
-      newProject: {
-        name: projectInf.name,
-        dsescription: projectInf.dsescription
-      }
-    });
+    const projectId = this.props.match.params.projectId;
+    if (Number.isInteger(parseInt(projectId))) {
+      const projectInf = require("./../../commonComponents/TableMember/projects.json")[0];
```

Comment:
@IsraaSulaiman I think for organizing the code, it's best if the imports are all at the top of the file, together - I'd recommend moving this one up to the top of the file (and then still using it here).